### PR TITLE
fix: UI tests

### PR DESCRIPTION
**Pull request by afeld**
Fixes https://github.com/cal-itp/benefits/issues/724. Builds on https://github.com/cal-itp/benefits/pull/1263.

This pull request does a few things:

- Removes the Cypress tests that didn't seem that useful (just checking text)
- Gets the remaining tests passing
- Removes the distinction between the "feature" and "UI" Cypress tests, as I don't think it was useful

### CODE REVIEW

File: .github/workflows/tests-cypress.yml [hunk 1]
```diff
@@ -18,12 +18,12 @@ jobs:
...
           docker compose up --detach client server
 
-      - name: Run feature tests
+      - name: Run Cypress tests
         uses: cypress-io/github-action@v5
         env:
           CYPRESS_baseUrl: http://localhost:8000
         with:
-          command: npm run cypress:feature
+          command: npm test
```
Comment (afeld) [inline .github/workflows/tests-cypress.yml:26]: The change that got them all running again.

**Comment by afeld** [comment]
Drat, the diff isn't picking up the renamed files. May be easier to review commit by commit.

File: tests/cypress/specs/eligibility-no-auth.cy.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+describe("Multiple Verifier, no AuthProvider: Verifier selection page spec", () => {
+  beforeEach(() => {
+    cy.visit("/");
+    helpers.selectAgency();
+  });
+
+  it("User sees two radio buttons", () => {
+    cy.get("input:radio").should("have.length", 2);
+    cy.contains("Courtesy Card");
```
Comment (thekaveman) [inline tests/cypress/specs/eligibility-no-auth.cy.js:13]: Would this same type of `.contains()` work to simplify the [`selectCourtesyCard` helper](https://github.com/cal-itp/benefits/pull/1277/files#diff-88f268da70bf80004fffc1b39325c8693737f2269ff8a95b808ff78ffc15192eR20)? 

I guess I'm not clear on why the text "Courtesy Card" is used here, but the position of the radio button in the list is used in the other case?

Comment (afeld) [reply tests/cypress/specs/eligibility-no-auth.cy.js:13]: Unfortunately it's not quite that simple. Here, it's simply checking whether the text is on the page, whereas to actually select the radio button, it needs to find the right `<input>`. Given that `Courtesy Card` appears as only part of the text in a `<span>` in the `<label>`, the selection isn't quite as simple.

Screenshot 2023-02-16 at 5 17 16 PM

**Comment by thekaveman** [comment]
I temporarily removed the branch protection requirement for Cypress tests, since it was using the old check name (`tests-ui`) that is now changed with this PR. 

Once this is merged we can re-add the protection requirement using the new check name (`tests-cypress`).

**Review by thekaveman — APPROVED** [review]
I'm good with re-combining these into a single type of test. We had it that way originally, then split out the feature tests as they weren't working as well and were taking a long time to run comparatively. 

But with the simpler startup using compose and general refocus of Cypress testing, this makes a lot of sense.

**Comment by thekaveman** [comment]
Updated branch protection rules for `dev`, `test`, and `prod` to require the newer `tests-cypress` check over the older one.

**Pull request merged**